src/main/java/org/jenkinsci/plugins/credentialsbinding/MultiBinding.java

```diff
@@ -62,7 +62,7 @@ protected MultiBinding(String credentialsId) {
     }
 
     /** Type token. */
-    protected abstract Class<C> type();
+    public abstract Class<C> type();
```

Comment:
Binary-incompatible for external implementations, I think.

Probably solvable by adding a `public final` method to `MultiBinding` to set up the tracking.